src/main/java/com/hedvig/backoffice/graphql/resolvers/ClaimResolver.kt

```diff
@@ -79,9 +79,11 @@ class ClaimResolver(
     }
 
     fun getType(claim: Claim, env: DataFetchingEnvironment?): Any? {
-        return if (claim._type == null) {
+        val claimType = claim._type
```

Comment:
You can avoid the first `if` and the nested `else when` by simply chaing this line to this:
```suggestion
        val claimType = claim._type ?: return null // now claimType will not be null on the next line
```